egs/libritts/tts1/run.sh

```diff
@@ -0,0 +1,250 @@
+win_length="" # window length
+
+# config files
+train_config=conf/train_pytorch_tacotron2+spkemb.yaml
+decode_config=conf/decode.yaml
+
+# decoding related
+model=model.loss.best
+griffin_lim_iters=1000  # the number of iterations of Griffin-Lim
+
+# Set this to somewhere where you want to put your data, or where
+# someone else has already put it. You'll want to change this
+# if you're not on the CLSP grid.
+datadir=/export/a15/vpanayotov/data
+
+# base url for downloads.
+data_url=www.openslr.org/resources/60
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train_clean_460
+dev_set=dev_clean
+eval_set=test_clean
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    echo "stage -1: Data Download"
+    mkdir -p ${datadir}
+    for part in dev-clean test-clean train-clean-100 train-clean-360; do
+        local/download_and_untar.sh ${datadir} ${data_url} ${part}
+    done
+fi
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    ### Task dependent. You have to make data the following preparation part by yourself.
+    ### But you can utilize Kaldi recipes in most cases
+    echo "stage 0: Data preparation"
+    for part in dev-clean test-clean train-clean-100 train-clean-360; do
+        # use underscore-separated names in data directories.
+        local/data_prep.sh ${datadir}/LibriTTS/${part} data/${part//-/_}
+    done
+fi
+
+feat_tr_dir=${dumpdir}/${train_set}; mkdir -p ${feat_tr_dir}
+feat_dt_dir=${dumpdir}/${dev_set}; mkdir -p ${feat_dt_dir}
+feat_ev_dir=${dumpdir}/${eval_set}; mkdir -p ${feat_ev_dir}
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    ### Task dependent. You have to design training and dev name by yourself.
+    ### But you can utilize Kaldi recipes in most cases
+    echo "stage 1: Feature Generation"
+
+    fbankdir=fbank
+    for x in dev_clean test_clean train_clean_100 train_clean_360; do
+        make_fbank.sh --cmd "${train_cmd}" --nj ${nj} \
+            --fs ${fs} \
+            --fmax "${fmax}" \
+            --fmin "${fmin}" \
+            --n_fft ${n_fft} \
+            --n_shift ${n_shift} \
+            --win_length "${win_length}" \
+            --n_mels ${n_mels} \
+            data/${x} \
+            exp/make_fbank/${x} \
+            ${fbankdir}
+    done
+
+    utils/combine_data.sh data/${train_set}_org data/train_clean_100 data/train_clean_360
+    utils/combine_data.sh data/${dev_set}_org data/dev_clean
+
+    # remove utt having more than 3000 frames
+    # remove utt having more than 400 characters
+    remove_longshortdata.sh --maxframes 3000 --maxchars 400 data/${train_set}_org data/${train_set}
+    remove_longshortdata.sh --maxframes 3000 --maxchars 400 data/${dev_set}_org data/${dev_set}
+
+    # compute global CMVN
```

Comment:
I know this is a copy-and-paste, but I'd suggest rephrasing. When I first read the comment (in #752) I was very confused because
- mel-spectrogram is used for decoder target in Tacotron. It's not cepstrum.
- "compute global CMVN (cesptrum mean-variance normalization) " doesn't really say what is done here, though I can understand. "compute statistics for CMVN" is more correct and easy to understand for me.

So what about `compute statistics for global CMVN`?  or more accurately, `compute statistics for global mean-variance normalization`?

Reply:
We are too familiar with kaldi...

Reply:
I agree with @r9y9.
It's not easy to change the file name, but just try to change the comment part.

Reply:
Changed to `compute statistics for global mean-variance normalization`.